### PR TITLE
Task 2.1+Task 2.2 and Task3 done

### PR DESCRIPTION
**Pull request by anna2506**
Выполнила задание 2.1, создала и натренировала FullyConnectLayer нейронную сеть.
Получила следующие результаты на финальном задании:
best validation accuracy achieved: 0.716000
Neural net test set accuracy: 0.676000

### CODE REVIEW

**Comment by anna2506** [comment]
@toshiks нет возможности добавить тебя как ревьюера

**Comment by anna2506** [comment]
В результате выполнения познакомилась с функциями Pytorch и в конце получила точность на тестовой выборке 0.8098.

**Comment by anna2506** [comment]
Заресолвила конфликт, смержила мастер enjoy-the-science в свой форк

**Comment by anna2506** [comment]
В первой части задания реализовала свёрточную нейронную сеть и подобрала параметры для переобучения сети за 50 эпох.
Во второй части поработала с PyTorch и GPU, реализовала LeNet model и затем реализовала свою сеть с лучшими параметрами, на финальном тесте точность на тестовых данных составила 82%, больше, чем получилось на модели LeNet, что и требовалось по заданию
